Describe breaking change. (#10033)
flutter/website: src/release/breaking-changes/pageview-controller.md

---
title: Make PageView.controller nullable
description: >-
  PageView.controller converted to be nullable.
---

## Summary

> Now, in consistency with other widgets, if controller is
> not provided in the widget constructor, the member `controller` is `null`.

Reviewer: ```suggestion
If a controller isn't provided in the constructor, the `controller` member is `null`.
This makes `PageviewController` consistent with other widgets.
```
